smoke/IotEdgeQuickstart/details/Details.cs

```diff
@@ -365,7 +365,12 @@ string BuildImageName(string name)
             ""settings"": {
               ""image"": ""<image-edge-hub>"",
               ""createOptions"": ""{\""HostConfig\"":{\""PortBindings\"":{\""8883/tcp\"":[{\""HostPort\"":\""8883\""}],\""443/tcp\"":[{\""HostPort\"":\""443\""}]}},\""Env\"":[\""SSL_CERTIFICATE_PATH=/mnt/edgehub\"",\""SSL_CERTIFICATE_NAME=edge-hub-server.cert.pfx\""]}""
-            }
+            },
+		    ""env"": {
+				""OptimizeForPerformance"": {
+					""value"": ""false""
```

Comment:
How hard will it be to make this value configurable? Preferably with a default? I am a bit concerned that the mainline path is not getting tested because of this change.

Reply:
It's not that hard, I will add a task for it.